frontend/src/Editor/LeftSidebar/sidebar-datasources.js

```diff
@@ -4,20 +4,20 @@ import { LeftSidebarItem } from './sidebar-item';
 import { DataSourceManager } from '../DataSourceManager';
 import { DataSourceTypes } from '../DataSourceManager/DataSourceTypes';
 
-export const LeftSidebarDataSources = ({ appId, darkMode, dataSources= [], dataSourcesChanged }) => {
-  const [open, trigger, content] = usePopover(false)
+export const LeftSidebarDataSources = ({ appId, darkMode, dataSources = [], dataSourcesChanged }) => {
+  const [open, trigger, content] = usePopover(false);
   const [showDataSourceManagerModal, toggleDataSourceManagerModal] = React.useState(false);
   const [selectedDataSource, setSelectedDataSource] = React.useState(null);
 
-  const renderDataSource = dataSource => {
+  const renderDataSource = (dataSource) => {
     const sourceMeta = DataSourceTypes.find((source) => source.kind === dataSource.kind);
     return (
       <tr
         role="button"
-        key={sourceMeta.kind.toLowerCase()}
+        key={sourceMeta.id}
```

Comment:
`sourceMeta` is just a metadata object, there won't be an `id` key. Can you please check?

Reply:
Yep, that's right, this was complaining on the console saying that 2 elements have same key, when 2 datasources of same kind are added, I have changed the key to index now